reporters/wdio/TerraWDIOTestDetailsReporter.js

```diff
@@ -56,7 +56,7 @@ class TerraWDIOTestDetailsReporter extends events.EventEmitter {
   }
 
   latestScreenshot(screenshotPath) {
-    this.screenshots.push(screenshotPath.screenshotPath);
+    if (!this.screenshots.includes(screenshotPath.screenshotPath)) this.screenshots.push(screenshotPath.screenshotPath);
```

Comment:
does our linter not enforce this?

```suggestion
    if (!this.screenshots.includes(screenshotPath.screenshotPath)) {
      this.screenshots.push(screenshotPath.screenshotPath);
    }
```

Reply:
no it didn't, I will change it